spec/fixtures/puppetdb/ssl-setup.sh

```diff
@@ -1,7 +1,10 @@
-#!/bin/bash
+#!/bin/sh
 
 ssl_command="puppetdb ssl-setup"
 
+# If CERTNAME environment variable is set use that, otherwise use HOSTNAME
+mycertname="${CERTNAME:-$HOSTNAME}"
```

Comment:
This is the upstream change that we intend to make. For now it is just here for testing. Should be removed once available.